gravitee-apim-rest-api/gravitee-apim-rest-api-model/src/main/java/io/gravitee/rest/api/model/api/RollbackApiEntity.java

```diff
@@ -108,7 +105,6 @@ public class RollbackApiEntity {
     @Schema(description = "API's gravitee definition version")
     private String graviteeDefinitionVersion;
 
-    @DeploymentRequired
```

Comment:
are you sure that modifying the flow mode does not require a deployment ?

Reply:
yes, it was interpreted nowhere, but maybe I'm worng

Reply:
After our discussion, you're right, it can be removed